spec/unit/block_score/util_spec.rb

```diff
@@ -1,113 +1,111 @@
 
-      context 'person' do
-        let(:input) { 'person' }
-        let(:output) { 'people' }
+      it_behaves_like 'to_plural'
+    end
 
-        it_behaves_like 'to_plural'
-      end
+    context 'watchlist_hit' do
+      let(:input) { 'watchlist_hit' }
+      let(:output) { 'watchlist_hits' }
 
-      context 'watchlist_hit' do
-        let(:input) { 'watchlist_hit' }
-        let(:output) { 'watchlist_hits' }
+      it_behaves_like 'to_plural'
+    end
+  end
 
-        it_behaves_like 'to_plural'
-      end
+  describe '.to_camelcase' do
+    shared_context 'to_camelcase' do
+      let(:method) { :to_camelcase }
+      include_context 'a transform'
     end
 
-    describe '.to_camelcase' do
-      shared_context 'to_camelcase' do
-        let(:method) { :to_camelcase }
-        include_context 'a transform'
-      end
+    context 'screaming_snake_case' do
+      let(:input) { 'screaming_snake_case' }
+      let(:output) { 'ScreamingSnakeCase' }
 
-      context 'screaming_snake_case' do
-        let(:input) { 'screaming_snake_case' }
-        let(:output) { 'ScreamingSnakeCase' }
+      it_behaves_like 'to_camelcase'
+    end
 
-        it_behaves_like 'to_camelcase'
-      end
+    context 'snake_case' do
+      let(:input) { 'snake_case' }
+      let(:output) { 'SnakeCase' }
 
-      context 'snake_case' do
-        let(:input) { 'snake_case' }
-        let(:output) { 'SnakeCase' }
+      it_behaves_like 'to_camelcase'
+    end
 
-        it_behaves_like 'to_camelcase'
-      end
+    context 'foo' do
+      let(:input) { 'foo' }
+      let(:output) { 'Foo' }
 
-      context 'foo' do
-        let(:input) { 'foo' }
-        let(:output) { 'Foo' }
+      it_behaves_like 'to_camelcase'
+    end
 
-        it_behaves_like 'to_camelcase'
-      end
+    context 'empty string' do
+      let(:input) { '' }
+      let(:output) { '' }
 
-      context 'empty string' do
-        let(:input) { '' }
-        let(:output) { '' }
+      it_behaves_like 'to_camelcase'
+    end
+  end
 
-        it_behaves_like 'to_camelcase'
-      end
+  describe '.parse_json' do
+    shared_context 'parse_json' do
+      let(:method) { :parse_json }
+      include_context 'a transform'
     end
 
-    describe '.parse_json' do
-      shared_context 'parse_json' do
-        let(:method) { :parse_json }
-        include_context 'a transform'
-      end
+    context 'valid input' do
+      let(:input) { output.to_json }
+      let(:output) { { foo: 'bar', baz: 'bat' } }
 
-      context 'valid input' do
-        let(:input) { output.to_json }
-        let(:output) { { foo: 'bar', baz: 'bat' } }
+      it_behaves_like 'parse_json'
+    end
 
-        it_behaves_like 'parse_json'
+    context 'translates errors' do
+      let(:error) do
+        'An error has occurred. ' \
+          'If this problem persists, please message support@blockscore.com.'
       end
+      let(:input) { '{{' }
+      subject(:parse_attempt) { -> { described_class.parse_json(input) } }
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/68/files#r52838287 (although it's not a shared spec, the same suggestion applies)